File: C++/프로그래머스/Lv.2/프로그래머스_lv2/프로그래머스_lv2/프린터.cpp
//#include<bits/stdc++.h>
//using namespace std;
//
//int solution(vector<int> priorities, int location) {
//    vector<int>vc;
//    map<int, int> mp;
//    for (size_t i = 0; i < priorities.size(); i++)
//    {
//        mp[priorities[i]] = i;
//    }
//    for (auto it : mp) {
//        vc.push_back(it.second);
//    }
//    reverse(vc.begin(), vc.end());
//    return vc[location];
//}
//int main() {
//    vector<int> pri = { 2, 1, 3, 2 };
//    int loc = 2;
//    cout<<solution(pri, loc);
//}

File: py/프로그래머스/lv.2/프린터/프린터/프린터.py

def solution(priorities, location):
    p = []
    for i in range(0,len(priorities)):
        p.append([priorities[i],i+1])
    p.sort(reverse=True)
    print(p)
    answer = p[location][1]
    return answer
p=[2, 1, 3, 2]	
l=2
print(solution(p,l))